### PR TITLE
Docs improvements

### PR DESCRIPTION
**Pull request by glyn**

### CODE REVIEW

File: pkg/knative/commands/deployer_create.go [hunk 1]
```diff
@@ -249,7 +249,9 @@ Create a Knative deployer.
 
 Build references are resolved within the same namespace as the deployer. As the
 build produces new images, the image will roll out automatically. Image based
-deployers must be updated manually to roll out new images.
+deployers must be updated manually to roll out new images. Rolling out an image
+means creating a Knative configuration with a pod referencing the image and a
```
Comment (scothis) [inline pkg/knative/commands/deployer_create.go]:
```suggestion
means creating a Knative Configuration with a pod referencing the image and a
```

File: pkg/knative/commands/deployer_create.go [hunk 2]
```diff
@@ -249,7 +249,9 @@ Create a Knative deployer.
...
+Knative route referencing the configuration.
```
Comment (scothis) [inline pkg/knative/commands/deployer_create.go]:
```suggestion
Knative Route referencing the Configuration.
```

File: pkg/knative/commands/deployer_create.go [hunk 3]
```diff
@@ -249,7 +249,9 @@ Create a Knative deployer.
 
 Build references are resolved within the same namespace as the deployer. As the
 build produces new images, the image will roll out automatically. Image based
-deployers must be updated manually to roll out new images.
+deployers must be updated manually to roll out new images. Rolling out an image
+means creating a Knative Configuration with a pod referencing the image and a
```
Comment (trisberg) [inline pkg/knative/commands/deployer_create.go]: should "pod" be "pod spec" or "PodSpec"?

Comment (glyn) [reply pkg/knative/commands/deployer_create.go]: I think there was a move to change from a Pod to PodSpec, but either way, I think we can keep this text informal and just say "pod".

**Pull request merged**